libs/pages/database/src/lib/ui/page-general/page-general.tsx

```diff
@@ -0,0 +1,54 @@
+import { DatabaseEntity, LoadingStatus } from '@console/shared/interfaces'
+import { BaseLink, HelpSection, Skeleton } from '@console/shared/ui'
+import About from '../about/about'
+import InstancesTable from '../instances-table/instances-table'
+
+export interface PageGeneralProps {
+  database?: DatabaseEntity
+  listHelpfulLinks: BaseLink[]
+  loadingStatus?: LoadingStatus
+  commitDeltaCount?: number
```

Comment:
This prop is not needed for database I think

Reply:
done